src/Type.cpp

```diff
@@ -6,7 +6,6 @@ namespace Halide {
 
 using std::ostringstream;
 
-namespace {
 uint64_t max_uint(int bits) {
```

Comment:
These three functions could probably be `constexpr` -- wouldn't help in the case added to CodegenX86 but might move the needle elsewhere?

Reply:
Done!